tests/ui/conftest.py

```diff
@@ -1,20 +1,50 @@
 import datetime
 import json
 import os
+import random
 import urlparse
 
 import jwt
 import pytest
 import requests
 from django.core.management import call_command
 from fxapom.fxapom import DEV_URL, PROD_URL, FxATestAccount
-from olympia.amo.tests import create_switch
+from olympia import amo
+from olympia.addons.forms import icons
+from olympia.addons.models import AddonUser, Preview
+from olympia.addons.utils import generate_addon_guid
+from olympia.amo.tests import (
+    addon_factory,
+    create_switch,
+    user_factory,
+    version_factory,
+)
+from olympia.constants.applications import APPS, FIREFOX
+from olympia.constants.base import (
+    ADDON_EXTENSION,
+    ADDON_PERSONA,
+    STATUS_PUBLIC
+)
+from olympia.landfill.collection import generate_collection
+from olympia.landfill.generators import generate_themes
+from olympia.reviews.models import Review
+from olympia.users.models import UserProfile
 from pytest_django import live_server_helper
 
+# TODO: Change imports to allow running tests against deployed instances.
+
 
 @pytest.fixture(scope='function')
-def my_base_url(base_url, request):
-    return base_url or request.getfixturevalue("live_server").url
+def my_base_url(base_url, request, pytestconfig):
+    """
+    Base URL used to start the 'live_server'.
+    """
+
+    if base_url:
+        pytestconfig.option.usingliveserver = False
+        return base_url or os.environ.get('PYTEST_BASE_URL')
```

Comment:
No need for `or os.environ.get('PYTEST_BASE_URL')` because `base_url` already has a value, and the pytest-base-url plugin already read from this environment variable.